fetch_maps/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package>
+  <name>fetch_maps</name>
+  <version>0.0.1</version>
+  <description>The fetch_maps package</description>
+
+  <maintainer email="ablasdel@fetchrobotics.com">Aaron Blasdel</maintainer>
+  <author>Michael Gregg</author>
+  <author>Michael Ferguson</author>
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <run_depend>slam_karto</run_depend>
```

Comment:
I put the build_map.launch file in here instead of in fetch_navigation so it does depend on it. 

I can move that out and then remove this.